dev/fattest.databases/resources/META-INF/services/org.testcontainers.dockerclient.DockerClientProviderStrategy

```diff
@@ -0,0 +1 @@
+componenttest.topology.utils.ExternalTestServiceDockerClientStrategy
```

Comment:
Is this file needed? I believe the copy in fattest.simplicity should be sufficient

Reply:
Removed.